README.md

```diff
@@ -11,7 +11,7 @@
 - Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
 - To get a local copy up and running follow these simple example steps
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
```

Comment:
https://github.com/Regiss05/To-do-list/blob/ebe115a1d4890cd0e5928d48ecf0b9d7c3148cef/src/index.js#L125-L130

- [ ] I have noticed some issues when I try to remove an item using the `x removing icon` from the todo list but when I reload the page all those removed items still show up. Kindly fix this and Make sure to reload the page to test the removed item is stored.